src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public class RightTriangle extends Figure {
+    private int base;
+    private int height;
+
+    {
+        name = "right triangle";
+    }
+
+    public RightTriangle(int base, int height, Color color) {
+        this.base = base;
+        this.height = height;
+        this.color = color.name().toLowerCase();
+    }
+
+    public int getBase() {
+        return this.base;
+    }
+
+    public int getHeight() {
+        return this.height;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString()
+                + "base: "
+                + getBase()
+                + " units, "
+                + "height: "
+                + getHeight()
+                + " units";
+    }
```

Comment:
Check the task requirenments

![image](https://github.com/mate-academy/jv-oop-advanced/assets/109764850/cf73b281-b206-4fe6-ae59-b2d2a67aeadb)

Reply:
use interface to add such behavior